src/components/tools/tool-tile.tsx

```diff
@@ -208,11 +208,13 @@ export class ToolTileComponent extends BaseComponent<IProps, IState> {
     const { ToolComponent, toolTileClass } = kToolComponentMap[model.content.type];
     const isPlaceholderTile = ToolComponent === PlaceholderToolComponent;
     const isTileSelected = ui.isSelectedTile(model);
+    const tileSelectedForComment = isTileSelected && ui.showChatPanel;
```

Comment:
This allows us to have multiple highlighted tiles, which doesn't seem quite right since only one can be commented upon, but since I don't have a better idea at the moment let's go with it for now and we can tweak it later once people have played with it.